goal_src/engine/target/target-part.gc

```diff
@@ -5,3 +5,9718 @@
+     :random-mult #x3f800000
+     )
+    (new 'static 'sp-field-init-spec
+     :field #x1c
+     :flags #x1
+     :initial-value #x415a740e
+     :random-range #x415a740e
+     :random-mult #x3f800000
+     )
+    (new 'static 'sp-field-init-spec
+     :field #x1f
+     :flags #x1
+     :initial-value -1039031628
+     :random-range #x4291a2b4
+     :random-mult #x3f800000
+     )
+    (new 'static 'sp-field-init-spec
+     :field #x20
+     :flags #x3
+     :initial-value -4
+     :random-mult 1
+     )
+    (new 'static 'sp-field-init-spec
+     :field #x24
+     :flags #x1
+     :initial-value #x3f5a740e
+     :random-mult #x3f800000
+     )
+    (new 'static 'sp-field-init-spec
+     :field #x2d
+     :flags #x1
+     :initial-value #x3f7ae148
+     :random-mult #x3f800000
+     )
+    (new 'static 'sp-field-init-spec
+     :field #x2e
+     :initial-value #x1c2
+     :random-mult 1
+     )
+    (new 'static 'sp-field-init-spec
+     :field #x2f
+     :initial-value #x4004
+     :random-mult 1
+     )
+    (new 'static 'sp-field-init-spec
+     :field #x32
+     :initial-value 15
+     :random-range 59
+     :random-mult 1
+     )
+    (new 'static 'sp-field-init-spec
+     :field #x33
+     :flags #x6
+     :initial-value #x7d8
+     )
+    (new 'static 'sp-field-init-spec :field #x43)
+    )
+   )
+  )
+
+;; failed to figure out what this is:
+(set!
+  (-> *part-id-table* 2008)
+  (new 'static 'sparticle-launcher
+   :init-specs
+   (new 'static 'inline-array sp-field-init-spec 2
+    (new 'static 'sp-field-init-spec
+     :field #x24
+     :flags #x1
+     :initial-value -1097751884
+     :random-mult #x3f800000
+     )
+    (new 'static 'sp-field-init-spec :field #x43)
+    )
+   )
+  )
+
+;; definition for function process-drawable-burn-effect
+;; Used lq/sq
+(defbehavior process-drawable-burn-effect target ((arg0 int))
+  (sound-play-by-name
+   (the-as
+    sound-name
+    (make-u128 #x6465 (-> (new 'static 'array uint64 1 #x6e7275622d746567) 0))
+    )
+   (new-sound-id)
+   1024
+   0
+   0
+   (the-as uint 1)
+   (the-as vector #t)
+   )
+  (let ((s5-1 (new 'stack 'rgbaf))
+        (s3-0 (-> *display* base-frame-counter))
+        (s4-1 (-> self parent))
+        )
+   (set!
+    (-> s5-1 quad)
+    (-> (the-as process-drawable (-> s4-1 0)) draw color-mult quad)
+    )
+   (let
+    ((s2-1
+      (vector-float*!
+       (the-as vector (new 'stack 'rgbaf))
+       (the-as vector s5-1)
+       0.0
+       )
+      )
+     )
+    (while (< (- (-> *display* base-frame-counter) s3-0) arg0)
+     (let ((v1-8 (- (-> *display* base-frame-counter) s3-0)))
+      (if (< v1-8 (/ arg0 2))
+       (vector-lerp!
+        (-> (the-as process-drawable (-> s4-1 0)) draw color-mult)
+        s5-1
+        s2-1
+        (/ (the float v1-8) (the float arg0))
+        )
+       (vector-lerp!
+        (-> (the-as process-drawable (-> s4-1 0)) draw color-mult)
+        s5-1
+        s2-1
+        (-
+         (-> (new 'static 'array float 1 1.0) 0)
```

Comment:
same here, this should probably be `["L...", "float", true]`.  The automatic labeling can't tell the difference between the two on files with far labels.